generate_num_for_draw.py

```diff
@@ -3,6 +3,7 @@
 
 a = string.ascii_lowercase

```

Comment:
Global variables must be written in capital letters. Try to name the variables so that their names reveal their meaning.

EX:
LETTERS= string.ascii_lowercase